trino-aws-proxy/src/main/java/io/trino/aws/proxy/server/rest/RequestLoggerController.java

```diff
@@ -71,10 +110,48 @@ public boolean isEnabled()
         }
     };
 
+    public record SaveEntry(String entryId, SigningServiceType serviceType, String message, Map<String, String> entries, Instant timestamp)
+    {
+        public SaveEntry
+        {
+            requireNonNull(serviceType, "serviceType is null");
+            requireNonNull(entryId, "entryId is null");
+            requireNonNull(message, "message is null");
+            entries = ImmutableMap.copyOf(entries);
+            requireNonNull(timestamp, "timestamp is null");
+        }
+    }
+
+    public enum EventType
+    {
+        REQUEST_START,
+        REQUEST_END,
+    }
+
+    public static String eventId(Instant timestamp, long requestNumber, EventType eventType)
+    {
+        int typeKey = switch (eventType) {
```

Comment:
Can we not log `REQUEST_START` and `REQUEST_END`? It may be more readable than `0` and `1`

Reply:
I want this to be sortable. It's not meant to carry information.